docs/DeveloperGuide.md

```diff
@@ -1539,9 +1588,28 @@ testers are expected to do more *exploratory* testing.
 
 ### Managing Lessons
 
+#### Editing a lesson
+
+1. Editing a lesson while all lessons are displayed
```

Comment:
```suggestion
1. Editing a lesson while all lessons are displayed.
```

Reply:
I think most of existing instructions do not include a fullstop here?